projects/lib/src/form/form-wizard/wizard-overview/wizard-overview.component.scss

```diff
@@ -1 +1 @@
-@import 'node_modules/@baloise/design-system-components/src/styles/global.utilities.scss';
+@import '~@baloise/design-system-components/src/styles/global.utilities';
```

Comment:
new angular complains for somthing like this:
```
Warning: 'C:\Users\Public\dev\repos\mybaloise-business\dashboard-enterprise-client\node_modules\@baloise\design-system-next-components\src\styles\global.helpers.sass' imports '~@baloise/design-system-next-css/scss/baloise-design-sys
tem.helpers.sass' with a tilde. Usage of '~' in imports is deprecated.

````
maybe we should leave import with node_modules?